helper/TabManager.php

```
@@ -0,0 +1,62 @@
+<?php 
+namespace OmekaTheme\Helper;
+
+use Laminas\View\Helper\AbstractHelper;
+
+class TabManager extends AbstractHelper
+{
+    public function getResourcePageBlocks($resource) {
+        $regions = ['full_width_main', 'left', 'main', 'right'];
+        $view = $this->getView();
+        $tabLayout = $view->themeSetting('tab_navigation_layout');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$tabLayout'. (Unused Code Rules)

(UnusedLocalVariable)